doc/getting_started.rst

```diff
@@ -102,41 +110,76 @@ This loads Sphinx-Gallery as one of your extensions, the ellipsis
+  Header and table of contents tree. It which will serve as the welcome page for 
+  that gallery.
+* ``sg_execution_times.rst`` - execution time of all example ``.py`` files, 
+  summarised in table format (`original pull request on GitHub
+  <https://github.com/sphinx-gallery/sphinx-gallery/pull/348>`_).
+* ``images`` - directory containing images produced during execution of the 
+  example ``.py`` files (more details in :ref:`image_scrapers`) and thumbnail
+  images for the gallery.
+* A directory for each sub-directory in 'example_dirs'. Within each directory
+  will be the above and below listed files for that 'sub-gallery'.
+
+Additionally for EACH ``.py`` file, a file with the following suffix is 
+generated:
+
+* ``.rst`` - the rendered rST version of the ``.py`` file, ready for Sphinx 
+  to build.
+* ``.ipynb`` - to enable the user to download a Jupyter notebook version of the
+  example.
+* ``.py`` - to enable the user to download a ``.py`` version of the example.
+* ``.py.md5`` - a md5 hash of the ``.py`` file, used to determine if changes
+  have been made to the file and thus if new output files need to be generated.
+* ``_codeobj.pickle`` - used to identify function names and to which module 
+  they belong (more details in 
+  :ref:`sphx_glr_auto_examples_plot_function_identifier.py`)
+
+Additionally, two compressed ``.zip`` files containing all the ``.ipynb`` and 
+``.py`` files are generated.
+
+For more advanced configuration, see the :ref:`configuration` page.
 
 Add your gallery to the documentation
 -------------------------------------
 
-When you build your documentation, sphinx-gallery will automatically populate
-the folder specified in ``gallery_dirs`` above with Sphinx-ready rST.
-It will create an ``index.rst`` file in the root of each gallery folder that
-contains the rST for that gallery (in this example, it is ``gallery_dirs/index.rst``).
-You can add it to your Sphinx navbar, or embed it with an ``.. include::`` statement.
+The ``index.rst`` file generated for your gallery can be added to the table of 
+contents tree in the main Sphinx ``doc/index.rst`` file  or embedded in a 
+Sphinx source ``.rst`` file with an ``.. include::`` statement.
 
 Build the documentation
 -----------------------
 
-In your Sphinx documentation directory, (e.g., ``myproject/doc``) execute:
+In your Sphinx source directory, (e.g., ``myproject/doc``) execute:
 
 .. code-block:: bash
 
     $ make html
 
-This will start the build of your complete documentation including the examples
-gallery. Once a build is completed, all your examples outputs are cached.
+This will start the build of your complete documentation. Both 
+the Sphinx-Gallery output files described above and 
+the Sphinx output HTML (or another output format depending on the 
+`Sphinx 'builder'
+<https://www.sphinx-doc.org/en/master/man/sphinx-build.html>`_ selected) will
+be generated. Once a build is completed, all your examples' outputs are cached.
```

Comment:
I would use `all the outputs from your examples will be cached.` I find `'` hard for none English speakers.